python/pyspark/mllib/util.py

```diff
@@ -40,7 +40,7 @@ def _parse_libsvm_line(line, multiclass=None):
         nnz = len(items) - 1
         indices = np.zeros(nnz, dtype=np.int32)
         values = np.zeros(nnz)
-        for i in xrange(nnz):
+        for i in range(nnz):
```

Comment:
Can we use the following instead? Using `range` would hurt performance in Python 2 here.

~~~
if sys.version >= '3':
    xrange = range
~~~